common/on_action/ml_monthly_country.txt

```diff
@@ -1,5 +0,0 @@
-﻿monthly_country_pulse = {
```

Comment:
this is a monthly pulse not specific from code - need to undelete

Reply:
I think I deleted a wrong file here yeah